zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -504,22 +504,20 @@ The ZooKeeper service can be monitored in one of three primary ways:
 ### Logging
 
 ZooKeeper uses **[SLF4J](http://www.slf4j.org)**
-version 1.7.5 as its logging infrastructure. For backward compatibility it is bound to
-**LOG4J** but you can use
-**[LOGBack](http://logback.qos.ch/)**
-or any other supported logging framework of your choice.
-
-The ZooKeeper default *log4j.properties*
-file resides in the *conf* directory. Log4j requires that
-*log4j.properties* either be in the working directory
+version 1.7.5 as its logging infrastructure. By default ZooKeeper is shipped with
```

Comment:
1.7.30?

Reply:
I'd recommend being less tightly coupled to the current patch version in this document. Just "1.7" should suffice here.

Reply:
This is done.